script.js

```diff
@@ -15,13 +20,10 @@ function randInRange(min, max) {
 	var range = max-min;
 	return Math.floor(Math.random()*range)+min;
 }
-
 // write a function "setStyles" that accepts an element and sets all numerical CSS properties to random values within logically derived ranges.
 function setStyles(element) {
-	var windowWidth = window.innerWidth,
-		windowHeight = window.innerHeight,
-		divWidth = randInRange(1,600),
-		divHeight = randInRange(1,500),
+	var divWidth = randInRange(1,windowWidth),
+		divHeight = randInRange(1,windowHeight),
```

Comment:
This opens the door for some massive boxes! Maybe want to check for the lesser of 500 or the width/height?